nx.json

```diff
@@ -68,7 +68,8 @@
       "ability-aave",
       "ability-debridge",
       "policy-send-counter",
-      "ability-sol-transaction-signer"
+      "ability-sol-transaction-signer",
```

Comment:
what's `ability-sol-transaction-signer` ? is that a solana signer?  if so, why do we need that in this PR?